js/libs/keycloak-admin-client/src/utils/fetchWithError.ts

```diff
@@ -42,3 +45,20 @@ export async function parseResponse(response: Response): Promise<any> {
 
   return data;
 }
+
+
+function getNetworkErrorMessage(data: unknown): string {
```

Comment:
I think `getErrorMessage` will suffice here.
```suggestion
function getErrorMessage(data: unknown): string {
```